client/components/tinymce/plugins/simple-payments/dialog/form.jsx

```diff
@@ -99,14 +99,14 @@ class ProductForm extends Component {
 							<FormInputValidation isError text={ translate( 'Invalid price' ) } /> }
 					</FormFieldset>
 					<FormFieldset>
-						<FormToggle
+						<CompactFormToggle
```

Comment:
Oh, I wish that `FormToggle` just had a `compact` attribute, like `Button` does. Adding a todo to my todolist :)